### ISSUE TITLE
Users not able to deleted their own views

### ISSUE THREAD

**Issue by mwl** [humio/issues#75]
Regular users do not seem to have the "Retention" tab in settings, which prevents them from deleting views they have created themselves.

* Humio Version
  1.4.9--build-5391--sha-2956460ce
* Screenshots
  ![image001](https://user-images.githubusercontent.com/81906/54357449-6234ac00-465e-11e9-96c4-3c8f3d9bc4bc.png)

**Comment by lieberkind** [comment]
Furthermore, we get an error when trying to create views on top of the sandbox repository in static-onprem mode. This is not a problem in Cloud.

![Screenshot 2019-03-15 at 09 02 21](https://user-images.githubusercontent.com/486949/54416939-20107680-4701-11e9-898e-5d1c4f1f5768.png)

**Comment by lieberkind** [comment]
As far as I can tell the issue lies on the backend. It returns `false` for the `deleteDataspace` permission, even though the user has created the view themselves. I think the correct behavior would be for the backend to return `true` when the user is the owner of the view.

**Comment by mortengrouleff** [comment]
I don't think we should do any work on this until we have completed the new role-based permissions UI, and removed the "members of repo" concept. Doiunf it before that point will introduce new permissions bugs in the backend I think
